### PR TITLE
[RFR] Redux without profanity - step 2 (Follow #105)

### PR DESCRIPTION
**Pull request by djhi**
Simplified entities sagas

### CODE REVIEW

File: src/frontend/js/app/entities/fetchEntities.spec.js [hunk 1]
```diff
@@ -21,62 +25,31 @@ describe('fetchEntitiesFactory', () => {
...
             headers: {
                 Accept: 'application/json',
-                Authorization: 'token',
                 'Content-Type': 'application/json; charset=utf-8',
             },
             credentials: 'include',
         });
     });
 
-    it('should handle failed response', done => {
```
Comment (fzaninotto) [inline src/frontend/js/app/entities/fetchEntities.spec.js]: why did you remove this one?

**Comment by fzaninotto** [comment]
👍

File: src/frontend/js/app/entities/fetchEntities.spec.js (hunk 1 above)
Comment (djhi) [inline src/frontend/js/app/entities/fetchEntities.spec.js]: I didn't, but the diff is misleading. You can find it a bit below

**Comment by djhi** [comment]
I'll do a final renaming/moving of some files after #108. The files in `src/frontend/js/app` should be at the frontend project root and those in entities are not targeting entities specifically anymore.
